content/flutter-testing/app-preview.md

```diff
@@ -8,7 +8,7 @@ aliases:
 Codemagic allows you to launch your iOS or Android app on an iOS simulator or Android emulator running in the browser, regardless of the operating system you are using. Test your app against different device and OS configurations, emulate GPS location or demo the latest app version without needing access to a physical device. 
 
 {{<notebox>}}
-This feature is available for **teams** on request. Please [contact us](https://codemagic.io/contact/) for more information.
+This feature is available for **teams** on request. You can request access in the application.
```

Comment:
if a request form will be somewhere in team settings or elsewhere, maybe  it would be good to say where in the application. If it is going to be visible in the dashboard, then that's fine.